spec/features/collaborator_add_remove_spec.rb

```diff
@@ -1,15 +1,42 @@
 require 'rails_helper'
 
-feature 'collaborator add remove' do
+feature 'collaborator add remove', js:true do
   scenario 'add free user as collaborator to private wiki and edit'   do
-  	
+    user = create(:user, email: "test@example.org", password: "helloworld", tier: "premium")
+    other_user = create(:user, email: "free@example.org", password: "helloworld", tier: "free")
+    wiki = create(:wiki, user: user, level: "private")
+    collab = create(:collaboration, user: other_user, wiki: wiki)
+    login_as(other_user)
+    visit wiki_path(wiki)
+    click_link 'edit wiki'
+    expect(page).to have_content("Edit wiki")
   end
 
   scenario 'add user to wiki multiple times' do
+    user = create(:user, email: "test@example.org", password: "helloworld")
+    other_user = create(:user, email: "free@example.org", password: "helloworld")
+	wiki = create(:wiki, user: user)
+    collab = create(:collaboration, user: other_user, wiki: wiki)
+    login_as(user)
+    visit edit_wiki_path(wiki)
+    #Question: how to write this part of test for add collaborator form where it should be empty because there are only 2 users in DB
```

Comment:
Consider what experience you'd prefer for the user when they view the edit wiki form and no collaborators are available to be added.

- Would you want them to still see the email input field?
- Would you want to show them a message (e.g. no more collaborators can be added) alongside/instead of the input field?
- If you do want to show the email input field, do you want to show it as readonly so they can't fill it in?

Once we know the desired user experience we can figure out how to test it.